src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -213,9 +213,14 @@ bool CheckControlDataByCapabilities(
   for (; it != control_data.map_end(); ++it) {
     const std::string& request_parameter = it->first;
     if (message_params::kLightState == request_parameter) {
-      return CheckLightNameByCapabilities(
-          module_caps[strings::kSupportedLights],
-          control_data[request_parameter][0]);
+      bool is_light_param_valid = false;
+      auto light_data = control_data[request_parameter].asArray()->begin();
+      for (; light_data != control_data[request_parameter].asArray()->end();
+           ++light_data) {
+        is_light_param_valid = CheckLightNameByCapabilities(
```

Comment:
is_light_param_valid will be over write in the loop and the last one get returned?

Reply:
@yang1070 yes, you are right

Reply:
@yang1070 Sorry, it was an old commit, please look at c88c032